jest.config.js

```diff
@@ -75,13 +80,15 @@ module.exports = {
         '!**/*.unit.test.{js,mjs}'
       ],
 
-      // Browser test increased timeout (5s to 15s)
-      testTimeout: 15000,
-
       // Web server and browser required
       globalSetup: './config/jest/browser/open.mjs',
       globalTeardown: './config/jest/browser/close.mjs'
     }
   ],
-  reporters: ['default', 'github-actions']
+
+  // Enable GitHub Actions reporter UI
+  reporters: ['default', 'github-actions'],
+
+  // Browser test increased timeout (5s to 15s)
+  testTimeout: 15000
```

Comment:
Jest no longer likes timeouts set at project-level. Fixes:

```console
● Validation Warning:

  Unknown option "testTimeout" with value 15000 was found.
  This is probably a typing mistake. Fixing it will remove this message.

  Configuration Documentation:
  https://jestjs.io/docs/configuration
```